var/spack/repos/builtin/packages/mii/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Mii(MakefilePackage):
+    """Mii is an inverted indexing system for environment modules.
+       After installation you will no longer need to load modules to run
+       applications.
+
+       To complete the installation into your shell, first load 'mii':
+           $ module load mii
+
+       Then, use the helper to install Mii to the shell:
+           $ mii install
+
+       Mii will then be available to all new shells."""
```

Comment:
`spack info mii` probably won't be formatted correctly.

Reply:
Oops — I’ll rewrite the description in a more standard way. Similar instructions are in the README so I’ll point there.